### PR TITLE
Doc clarifications

### LINKED ISSUE #2016: Incorrect packaged readme
Distributed readme says:

```
To load the Elasticsearch template and pipelines for APM Server into Kibana, run:

    ./apm-server setup -e
```

However, that command only configures the template, not the pipelines

### PR DESCRIPTION
**Pull request by jalvz**
fixes #2016 

also part of #2026

### CODE REVIEW

File: _meta/beat.yml [hunk 1]
```diff
@@ -188,7 +188,7 @@ apm-server:
 #setup.template.name: "apm-%{[observer.version]}"
 
 # Template pattern. By default the template pattern is "apm-%{[observer.version]}-*" to apply to the default index settings.
-# The first part is the version of the beat and then -* is used to match all daily indices.
+# The first part is the version of the apm-server and then -* is used to match all daily indices.
```
Comment (graphaelli) [inline _meta/beat.yml]: remove `the`

File: _meta/beat.yml [hunk 2]
```diff
@@ -230,7 +230,7 @@ apm-server:
 
 #================================ Outputs =====================================
 
-# Configure what output to use when sending the data collected by the beat.
+# Configure what output to use when sending the data collected by the apm-server.
```
Comment (graphaelli) [inline _meta/beat.yml]: another `the` removal

File: _meta/beat.yml [hunk 3]
```diff
@@ -29,7 +29,7 @@ apm-server:
   # Authorization token to be checked. If a token is set here the agents must
   # send their token in the following format: Authorization: Bearer <secret-token>.
   # It is recommended to use an authorization token in combination with SSL enabled,
-  # and save the token in the beats keystore.
+  # and save the token in the apm-server keystore.
```
Comment (graphaelli) [inline _meta/beat.yml]: this `the` is ok

File: _meta/beat.yml (hunk 1 above)
Comment (jalvz) [reply _meta/beat.yml]: amended

File: _meta/beat.yml (hunk 2 above)
Comment (jalvz) [reply _meta/beat.yml]: amended

**Pull request merged**